mybatis-plus/src/main/java/com/baeldung/mybatisplus/codegenerator/MyBatisPlusCodeGenerator.java

```diff
@@ -0,0 +1,20 @@
+package com.baeldung.mybatisplus.codegenerator;
+
+import com.baomidou.mybatisplus.generator.FastAutoGenerator;
+import com.baomidou.mybatisplus.generator.engine.FreemarkerTemplateEngine;
+
+public class MyBatisPlusCodeGenerator {
+
+    public static void main(String[] args) {
+        FastAutoGenerator.create("jdbc:h2:file:~/mybatisplus", "sa", "")
+            .globalConfig(builder -> {
+                builder.author("anshulbansal")
+                    .outputDir("../tutorials/mybatis-plus/src/main/java/");
+            })
+            .packageConfig(builder -> builder.parent("com.baeldung.mybatisplus.codegenerator"))
+            .templateEngine(new FreemarkerTemplateEngine())
+            .execute();
+
+    }
+
+}
```

Comment:
Don't include main methods in the PR, we use unit tests not main methods

Reply:
created the `generateCode` method. However, it can't be tested in the integration tests since when the code classes are generated by the test they are not seen in the classloader/files.